apps/camunda/camunda-api/ithc.yaml

```diff
@@ -7,3 +7,10 @@ spec:
     java:
       environment:
         WA_AUTO_CONFIGURE_TASKS_ENABLED: false
+        CAMUNDA_DB_HOST: "hmcts-camunda-v14-flexible-{{ .Values.global.environment }}.postgres.database.azure.com"
+        CAMUNDA_DB_USER_NAME: "camundaadmin"
```

Comment:
Are we planning to change the username from default postgresql user?

Reply:
The existing single servers for camunda use a similar username to this but with `@hmcts-camunda-$env`. The flexible module sets the username without the `@` but the inputs to the module are the same as before.